uqcsbot/minecraft.py

```diff
@@ -93,6 +93,54 @@ async def mcwhitelist(self, interaction: discord.Interaction, username: str):
 
         db_session.close()
 
+    @app_commands.command()
+    @app_commands.describe(username="Minecraft username to unwhitelist.")
+    @yelling_exemptor(input_args=["username"])
```

Comment:
I wouldn't bother adding the `@yelling_exemptor`, that just makes the command unusable in #yelling (it'll enforce that the username needs to be all-upper-case, which probably mucks with the actual whitelisting)